Library/Homebrew/cmd/log.rb

```diff
@@ -56,6 +56,6 @@ def git_log(cd_dir, path = nil, tap = nil)
     end
     args = ARGV.options_only
     args += ["--follow", "--", path] unless path.nil?
-    exec "git", "log", *args
+    system "git", "log", *args
```

Comment:
I assume there's a reason this is `system` instead of `safe_system`?

Reply:
Yeh, the default `git log` exit code on ordinary usage is non-zero.